api/osrd_infra/migrations/__init__.py

```diff
@@ -2,7 +2,24 @@
 from django.db import migrations, models
 
 
-def run_sql_complexe_add_foreign_key(table_name: str, field_name: str, model_name: str, link_model: str, link_table: str, nullable: bool = False):
+def run_sql_complexe_add_foreign_key(table_name: str, field_name: str, model_name: str, link_model: str,
+                                     link_table: str, nullable: bool = False, related_name: str = ""):
+    if related_name:
+        field = models.ForeignKey(
+            on_delete=django.db.models.deletion.CASCADE,
+            to=f"osrd_infra.{link_model}",
+            related_name=related_name,
+            null=nullable,
+            blank=nullable,
+        )
+    else:
+        field = models.ForeignKey(
+            on_delete=django.db.models.deletion.CASCADE,
+            to=f"osrd_infra.{link_model}",
+            null=nullable,
+            blank=nullable,
+        )
```

Comment:
We can do something like that:
```python
related_name = {"related_name": related_name}
field = models.foreignKey(
         on_delete=django.db.models.deletion.CASCADE,
         to=f"osrd_infra.{link_model}",
         null=nullable,
         blank=nullable,
         **related_name
)
```

Note that the `related_name` arg should be `None` by default.